client/my-sites/plugins/plugins-category-results-page/index.jsx

```diff
@@ -29,18 +29,19 @@ const PluginsCategoryResultsPage = ( { category, siteSlug, sites } ) => {
 
 	return (
 		<>
-			<Header title={ categoryName } count={ title } subtitle={ categoryDescription } />
-
-			<FullListView
+			<PluginsBrowserList
+				title={ categoryName }
+				subtitle={ categoryDescription }
+				resultCount={ resultCount }
 				plugins={ plugins }
 				listName={ category }
 				site={ siteSlug }
 				showPlaceholders={ isFetching }
-				sites={ sites }
+				currentSites={ sites }
```

Comment:
This has caused issues before, just noting this change is correct. `FullListView` called this `sites` but `PluginsBrowserList` calls it `currentSites`